wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -29,92 +29,94 @@ enum CoreDataMessagingMigrationVersion: String, CaseIterable {
-    case version2_112 = "zmessaging2.112.0"
-    case version2_111 = "zmessaging2.111.0"
-    case version2_110 = "zmessaging2.110.0"
-    case version2_109 = "zmessaging2.109.0"
-    case version2_108 = "zmessaging2.108.0"
-    case version2_107 = "zmessaging2.107.0"
-    case version2_106 = "zmessaging2.106.0"
-    case version2_105 = "zmessaging2.105.0"
-    case version2_104 = "zmessaging2.104.0"
-    case version2_103 = "zmessaging2.103.0"
-    case version2_102 = "zmessaging2.102.0"
-    case version2_101 = "zmessaging2.101.0"
-    case version2_100 = "zmessaging2.100.0"
-    case version2_99 = "zmessaging2.99.0"
-    case version2_98 = "zmessaging2.98.0"
-    case version2_97 = "zmessaging2.97.0"
-    case version2_96 = "zmessaging2.96.0"
-    case version2_95 = "zmessaging2.95.0"
-    case version2_94 = "zmessaging2.94.0"
-    case version2_93 = "zmessaging2.93.0"
-    case version2_92 = "zmessaging2.92.0"
-    case version2_91 = "zmessaging2.91.0"
-    case version2_90 = "zmessaging2.90.0"
-    case version2_89 = "zmessaging2.89.0"
-    case version2_88 = "zmessaging2.88.0"
-    case version2_87 = "zmessaging2.87.0"
-    case version2_86 = "zmessaging2.86.0"
-    case version2_85 = "zmessaging2.85.0"
-    case version2_84 = "zmessaging2.84.0"
-    case version2_83 = "zmessaging2.83.0"
-    case version2_82 = "zmessaging2.82.0"
-    case version2_81 = "zmessaging2.81.0"
-    case version2_80 = "zmessaging2.80.0"
+    case v118 = "zmessaging2.118.0"
+    case v117 = "zmessaging2.117.0"
+    case v116 = "zmessaging2.116.0"
+    case v115 = "zmessaging2.115.0"
+    case v114 = "zmessaging2.114.0"
+    case v113 = "zmessaging2.113.0"
+    case v112 = "zmessaging2.112.0"
+    case v111 = "zmessaging2.111.0"
+    case v110 = "zmessaging2.110.0"
+    case v109 = "zmessaging2.109.0"
+    case v108 = "zmessaging2.108.0"
+    case v107 = "zmessaging2.107.0"
+    case v106 = "zmessaging2.106.0"
+    case v105 = "zmessaging2.105.0"
+    case v104 = "zmessaging2.104.0"
+    case v103 = "zmessaging2.103.0"
+    case v102 = "zmessaging2.102.0"
+    case v101 = "zmessaging2.101.0"
+    case v100 = "zmessaging2.100.0"
+    case v99 = "zmessaging2.99.0"
+    case v98 = "zmessaging2.98.0"
+    case v97 = "zmessaging2.97.0"
+    case v96 = "zmessaging2.96.0"
+    case v95 = "zmessaging2.95.0"
+    case v94 = "zmessaging2.94.0"
+    case v93 = "zmessaging2.93.0"
+    case v92 = "zmessaging2.92.0"
+    case v91 = "zmessaging2.91.0"
+    case v90 = "zmessaging2.90.0"
+    case v89 = "zmessaging2.89.0"
+    case v88 = "zmessaging2.88.0"
+    case v87 = "zmessaging2.87.0"
+    case v86 = "zmessaging2.86.0"
+    case v85 = "zmessaging2.85.0"
+    case v84 = "zmessaging2.84.0"
+    case v83 = "zmessaging2.83.0"
+    case v82 = "zmessaging2.82.0"
+    case v81 = "zmessaging2.81.0"
+    case v80 = "zmessaging2.80.0"
 
     var nextVersion: Self? {
         switch self {
-        case .version2_116,
-                .version2_117:
```

Comment:
This was a bug (introduced [here](https://github.com/wireapp/wire-ios/pull/1347/files#diff-34674e147f7b2974213af0e23acb90b04a3ac3ed482fd2f9d316885d8be4fb09)) returning `nil` and not doing the migration from 116 to 117.
It got fixed now both versions migrating directly to 118 (automatic light weight):
```swift
case .v116, .v117:
    return .v118
```